roles/core/pxe_stack/molecule/default/verify.yml

```diff
@@ -0,0 +1,184 @@
+---
+- name: Verify
+  hosts: management1
+  tasks:
+
+    - name: Collect package facts
+      package_facts:
+        manager: auto
+
+    - name: Collect services facts
+      service_facts:
+
+    - name: Collect services of zone public
+      command: firewall-cmd --zone=public --list-all
+      register: firewall_cmd_result
+      changed_when: False
+
+    - name: Firewall zone public check presence of service http and tftp
+      assert:
+        that: firewall_cmd_result.stdout |
+                    regex_findall(('http|tftp'), multiline=False) | length == 2
+
+    - name: Assert pxe needed packages are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - atftp
+        - ipxe-x86_64-bluebanquise
+        - ipxe-arm64-bluebanquise
+        - httpd
+
+    - name: Assert pxe extra el8 packages are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - python3-policycoreutils
+        - clustershell
+        - python3-clustershell
+        - policycoreutils
+      when:
+        - ansible_facts.os_family == "RedHat"
+        - ansible_facts.distribution_major_version == "8"
+
+    - name: Assert pxe extra el7 packages are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - policycoreutils-python
+        - python36-PyYAML
```

Comment:
Why not [python36](https://github.com/bluebanquise/bluebanquise/blob/f43f8bd5da0085dde0404cb1e430cc79440289d0/roles/core/pxe_stack/vars/RedHat_7.yml#L10)?

Reply:
On centos7 the package python36 is not found.
```
[root@management1 /]# rpm -q python36
package python36 is not installed
```


When the package python36 is installed it matches the package python3.
We should check/install perhaps python3 instead of python36 ?

```
[root@management1 /]# yum whatprovides python36
Loaded plugins: fastestmirror, ovl
Loading mirror speeds from cached hostfile
 * base: centos.mirror.ate.info
 * epel: epel.mirror.omnilance.com
 * extras: centos.crazyfrogs.org
 * updates: centos.mirror.fr.planethoster.net
python3-3.6.8-13.el7.i686 : Interpreter of the Python programming language
Repo        : base
Matched from:
Provides    : python36 = 3.6.8-13.el7
```

Reply:
OK. Drop it, this is a dependency of python36-PyYAML, no need to test it. Thanks.